### PR TITLE
Replace `request.REQUEST` with `POST` and `GET`

### PR DESCRIPTION
**Pull request by yourcelf**
Addresses #558, but not in the most ideal way.

Django deprecated `request.REQUEST` in 1.7 and removed it in 1.8.  Developers are advised to ["Use the more explicit GET and POST instead."](https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.REQUEST)

This PR does not choose which of POST or GET to use in each of the previous invocations of `request.REQUEST`, instead, it just repeats the old functionality of `request.REQUEST` (check `POST` first, then check `GET`).  It would be more ideal to go through and pick one of POST or GET depending on what the method expects, or if it really needs to use both, to keep logic such as this PR uses.

In short: this PR keeps the same prior functionality, but does not improve it in the manner Django maintainers intended by deprecating `request.REQUEST`.

Also: removes the `load url from future` from the example view. The future is now, and url can no longer be loaded from it. :)

### CODE REVIEW

**Comment by tschellenbach** [comment]
Thanks, this sounds like the right approach for maintaining backwards compatibility.

**Comment by briansanchez** [comment]
replacing request.REQUEST with POST it worked for me..
